nomad/local/grapl-local-infra.nomad

```diff
@@ -199,9 +220,11 @@ job "grapl-local-infra" {
           type    = "script"
           name    = "check_kafka"
           command = "/bin/bash"
+          # Interpolated by bash, not nomad
```

Comment:
note: previously we were just saying `nc -vz localhost` since, in bash, `${var.KAFKA_BROKER_PORT}` doesn't evaluate to anything, and bash just silently lets variables default to ""